template/deploy/helm/[[operator]]/resources/default_node_resources.yaml.j2

```diff
@@ -0,0 +1,6 @@
+limits:
+  cpu: {[ operator.daemonset_resources.cpu_limit  | default(resource_defaults.cpu_limit) }]
```

Comment:
We're pretty inconsistent about referring to this as `node` or `daemonset` right now, we should at least pick one.